conan/cli/commands/graph.py

```diff
@@ -118,6 +119,8 @@ def graph_info(conan_api, parser, subparser, *args):
                            help='Print information only for packages that match the patterns')
     subparser.add_argument("-d", "--deployer", action="append",
                            help='Deploy using the provided deployer to the output folder')
+    subparser.add_argument("--output-deployer",
```

Comment:
I guess not ``--deployer-output`` or ``--deployer-folder`` to avoid conflicts/overlap with ``--deployer`` in ``argparse``?
Maybe something that contains ``folder`` is a bit more descriptive/intuitive?

Reply:
It works good with `--deployer`, but some tests (and users!) might still be using the old `--deploy` syntax, which _would_ conflict. Do we force users to not use `--deploy` anymore and fix the tests?